Resources/Prototypes/Entities/Structures/Wallmounts/switch.yml

```diff
@@ -149,3 +149,56 @@
         Left: []
         Right: []
         Middle: []
+
+#directional
+
+- type: entity
+  id: SignalSwitchDirectional
+  name: signal switch
+  suffix: directional
+  parent: SignalSwitch
+  description: It's a switch for toggling power to things.
```

Comment:
You should be able to leave name and description blank if they're the same as the parent. The two fields inherit.

Reply:
yep, ok